var/spack/repos/builtin/packages/py-bluepysnap/package.py

```diff
@@ -34,3 +34,9 @@ class PyBluepysnap(PythonPackage):
 
     depends_on('py-click@7.0:', type='run')
     depends_on('py-pathlib2@2.3:', type='run')
+
+    @run_after('install')
+    @on_package_attributes(run_tests=True)
+    def check_install(self):
+        Executable('bluepysnagh --help')()
```

Comment:
Hm, does that work?

Maybe this is better?
```suggestion
        Executable('bluepysnap')('--help')
```

Reply:
Because I couldn't run this locally on BB5, I decided to use CI to verify that it works. So I made this error intentionally, and hoped that CI would fail. CI didn't fail. My question is whether this code gets executed at all?

Reply:
Ah, you're right. Here, we don't install with `spack install --test=…`, so this does not get called.

But it should work on BB5 locally if you do `spack install --test=root py-bluepysnap`

Reply:
Well, my main hope was to run such tests in CI. When I tried to run it locally on BB5, I've got:
```
-bash-4.2$ spack install --test=all py-bluepy@0.14.14
==> Error: maximum recursion depth exceeded
```